src/main/java/nl/tudelft/ewi/devhub/server/database/entities/User.java

```diff
@@ -116,5 +114,23 @@ public boolean isPasswordMatch(String password) {
 		return password != null && this.password != null
 				&& BCrypt.checkpw(password, this.password);
 	}
+
+	public List<Notification> getNotifications() {
+		List<Notification> notifications = new ArrayList<Notification>();
```

Comment:
I think something like `Lists.transform(notificationsToUsersList, NotificationsToUser::getNotification)` could work too. No need to edit this.